.github/workflows/shared_deploy_exchange.yml

```diff
@@ -28,6 +28,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      # Remove when switching to the new Exchange - BUSINESS_GROUP_ID_GLOBAL_2 completely for all ENV (DEV, STAGING, PROD)
+      - name: Modify groupId in pom.xml
+        shell: bash
+        run: |
+          perl -0777 -pe 's/<groupId>b1493a71-3550-4d96-a623-e3f11d038f62<\/groupId>/<groupId>${{ secrets.BUSINESS_GROUP_ID }}<\/groupId>/' -i.bak pom.xml
+          cat pom.xml
+
```

Comment:
> ` completely for all ENV (DEV, STAGING, PROD)

So `BUSINESS_GROUP_ID_GLOBAL` will be removed/unused in the future? 💭 

Seems like it totally missed this in Slack threads.

Reply:
Yeah @toby-thanathip there was a plan to remove the `BUSINESS_GROUP_ID_GLOBAL` completely - all DEV, STAGING, PROD will be on the `BUSINESS_GROUP_ID_GLOBAL_2`